pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
     "pyaml_env",
     "pyresample",
     "h5netcdf",
-    "tensorstore",
+    "xarray-tensorstore",
```

Comment:
Im guessing we are now importing some bits and overwriting some bits? Trying to keep the code down?

Reply:
If so, is it worth pinning this to a version, incase the code gets changed, especialyl if we are import private functions